lib/lib-doi/pom.xml

```diff
@@ -95,5 +91,16 @@
 			<groupId>org.springframework</groupId>
 			<artifactId>spring-core</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>xerces</groupId>
+			<artifactId>xercesImpl</artifactId>
+			<version>2.12.2</version>
```

Comment:
maybe create a ticket to remove xerces in the future.  tag this as do not use this as it is slated for removal.